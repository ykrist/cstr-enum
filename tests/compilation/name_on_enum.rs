use cstr_enum::*;

#[derive(FromCStr, AsCStr)]
#[cstr(name="egg")]
enum Enum {
  A
}

fn main() {

}