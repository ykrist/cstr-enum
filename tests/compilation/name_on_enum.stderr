error: attribute must be placed on variants
 --> $DIR/name_on_enum.rs:4:1
  |
4 | #[cstr(name="egg")]
  | ^^^^^^^^^^^^^^^^^^^
